specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/examples/integrations/GetIntegrationGroups_example.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2023-07-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "integrationId": "ad9a8e26-29d9-4829-bb30-e597a58cdbb8",
+    "namePrefix": "Platform group"
+  },
+  "responses": {
+    "200": {
+      "body": [
+        {
+          "name": "Group1",
```

Comment:
I'd expect something like this: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-04-01/examples/NetworkSecurityGroupRuleCreate.json 

have a name, id, and a property bag with the description and email